RealmSwift/Tests/ObjectAccessorTests.swift

```diff
@@ -329,4 +329,30 @@ class ObjectAccessorTests: TestCase {
         object.optObjectCol = nil
         XCTAssertNil(object.optObjectCol)
     }
+
+    func testLinkingObjectsDynamicGet() {
+        let fido = SwiftDogObject()
+        let owner = SwiftOwnerObject()
+        owner.dog = fido
+        owner.name = "JP"
+        let realm = try! Realm()
+        try! realm.write {
+            realm.add([fido, owner])
```

Comment:
@austinzheng is it necessary to add both dog and owner to realm? in my example I had only adding the 'child' / linking object.

Reply:
It's not, and it doesn't produce a different outcome.